hpi-schul-cloud/schulcloud-server
# BC-6654 - Update mongo to v5

# Description

QA note:
Tested change one env: 
https://bc-6654-update-mongodb.nbc.dbildungscloud.dev/
No issues found. I have done routine tests, creation course, team, task with content, adding user, tested on accounts admin, teacher, student. I also checked versions and job in openlens. See:
https://ticketsystem.dbildungscloud.de/browse/BC-6654



## Links to Tickets or other pull requests

https://ticketsystem.dbildungscloud.de/browse/BC-6654
https://github.com/hpi-schul-cloud/dof_app_deploy/pull/839
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4916
https://github.com/hpi-schul-cloud/infra-tools/pull/88
https://github.com/hpi-schul-cloud/end-to-end-tests/pull/453
https://github.com/hpi-schul-cloud/docker-compose/pull/70
## Changes


## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
